manifests/init.pp

```diff
@@ -57,26 +57,36 @@
       SOFT_UID_DOMAIN = TRUE
       | EOF
     ,
-    require => Yumrepo['condor'],
+    require => Package['condor'],
   }
 
   file { '/etc/condor/config.d/schedd':
     ensure  => file,
-    content => 'DAEMON_LIST = MASTER, SCHEDD',
-    require => Yumrepo['condor'],
+    content => @(CONTENT)
+      DAEMON_LIST = MASTER, SCHEDD
+      HISTORY_CONTAINS_JOB_ENVIRONMENT = False
+      | CONTENT
+    ,
+    require => Package['condor'],
```

Comment:
While it isn't "wrong", the repetitive ordering of resources on the `condor` package is a sign that this class should be split up into multiple sub-classes and the ordering is then set on the sub-classes. E.g.

https://github.com/lsst-it/puppet-rke2/blob/master/manifests/init.pp#L47-L62

That type of refactoring should be possible without changes to the spec.